db/migrate/20170928215837_add_star_index_path_to_sample.rb

```diff
@@ -0,0 +1,5 @@
+class AddStarIndexPathToSample < ActiveRecord::Migration[5.1]
+  def change
+    add_column :samples, :s3_star_index_path, :text
```

Comment:
seems overkill to generate two migrations for this. You can just put both of these lines to one migration.